lib/designator/selection.ex

```diff
@@ -37,10 +37,16 @@ defmodule Designator.Selection do
         Designator.UserCache.add_recently_selected(user, selected_ids)
         selected_ids
       :nil ->
-        Rollbax.report(:throw, :selection_timeout, System.stacktrace(),
+        try do
+          # throw an arbitrary run time error
+          raise "the task didn't respond on time"
+        rescue
+          exception ->
```

Comment:
Just wondering, do we also want to do anything with the `exception` received?

Reply:
No really - we only raise this exception now to access the stacktrace for error reporting. It's quite arbitrary and contains no useful information to the actual report.

FWIW the error report is signalling the selector can't complete before the 1 s timeout. We want designator to be fast so we try to avoid these long running selection queries. We'd like to understand why some subject selection requests timeout and perhaps we modify the code or extend the timeout.